lib/Helper/Configs/next.config.1.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
```

Comment:
@roggenkemper Are these only used in tests? If so, can we put configs in  it in __tests__?